packages/data-context/src/data/coreDataShape.ts

```diff
@@ -8,6 +8,8 @@ import type { Server } from 'http'
 import type { ErrorWrapperSource } from '@packages/errors'
 import type { GitDataSource, LegacyCypressConfigJson } from '../sources'
 
+const nmi = require('node-machine-id')
```

Comment:
Should this be `import`?

Reply:
Good question - meant to circle back and check this when I copy-pasta'ed it over. Will check now

Reply:
Updated to `import`